### PR TITLE
kdiff3: initial version of the package

### PR DESCRIPTION
**Pull request by davydden**
~~blocked by https://github.com/LLNL/spack/issues/863 which is a requirement for `qt`~~

### CODE REVIEW

**Comment by davydden** [comment]
@tgamblin this PR is ready to review. Qt still fails to build on macOS, so I can't test it here.

File: var/spack/repos/builtin/packages/libxau/package.py [hunk 1]
```diff
@@ -34,11 +35,10 @@ class Libxau(Package):
     version('1.0.8', '685f8abbffa6d145c0f930f00703b21b')
 
     depends_on('xproto')
+    depends_on('pkg-config')
```
Comment (becker33) [inline var/spack/repos/builtin/packages/libxau/package.py]: Can you make this a build type dependency?

Comment (davydden) [inline var/spack/repos/builtin/packages/libxau/package.py]: @becker33 done.

**Comment by adamjstewart** [comment]
kdiff3 is missing the license at the top of the file.

**Comment by davydden** [comment]
> kdiff3 is missing the license at the top of the file

fixed.

**Comment by adamjstewart** [comment]
LGTM

**Pull request merged**